data/mods/Magiclysm/Spells/attunements/Sun_Mage.json

```diff
@@ -29,5 +29,27 @@
     "max_level": 35,
     "base_casting_time": 350,
     "energy_source": "MANA"
+  },
+  {
+    "id": "sunmage_pocket_sun",
+    "type": "SPELL",
+    "name": "Pocket Sun",
```

Comment:
are you _sure_ this spell is strong enough to take up one of the 2 spell slots for the attunement?